pkg/sdn/plugin/subnets.go

```diff
@@ -155,6 +165,37 @@ func (master *OsdnMaster) watchNodes() {
 	}
 }
 
+// Because openshift-sdn uses an overlay and doesn't need GCE Routes,
+// we need to clear the kubelet-added NetworkUnavailable condition
+// that kubelet adds to initial node status when using GCE.
+// TODO: make upstream kubelet more flexible with overlays and GCE
```

Comment:
So NetworkUnavailable doesn't get set when not using GCE? Or it gets cleared by some other means?